packages/block-library/src/group/edit.native.js

```diff
@@ -72,14 +89,34 @@ function GroupEdit( {
 }
 
 export default compose( [
-	withColors( 'backgroundColor' ),
 	withSelect( ( select, { clientId } ) => {
-		const { getBlock } = select( 'core/block-editor' );
+		const {
+			getBlock,
+			hasSelectedInnerBlock,
+			getSelectedBlockClientId,
+		} = select( 'core/block-editor' );
 
 		const block = getBlock( clientId );
+		const hasInnerBlocks = !! ( block && block.innerBlocks.length );
+		const isInnerBlockSelected =
+			hasInnerBlocks && hasSelectedInnerBlock( clientId );
+		let isLastInnerBlock = hasInnerBlocks;
```

Comment:
I'm a bit confused by this name. Does that determine if the last inner block is selected? If yes, then shouldn't it be `isLastInnerBlockSelected`?

Hmmm, what in the case when the block has inner blocks but none of them is selected? The `isLastInner` would be `true`. Is that correct?

Reply:
Thanks for the suggestions, I went ahead and changed the name and changed the default value to `false`